ui/pages/Public/components/Faq.jsx

```diff
@@ -466,15 +466,17 @@ const FAQS = [
   }, {
     [ENGLISH]: {
       header: 'Q. How do I add data to an existing project in seqr?',
-      content: `To add new data, create a new joint called VCF with all the samples you want in your seqr project, 
-      including those you had previously loaded, and upload it using the Load Additional Data feature on the Project 
-      Page. All notes and tags saved in previously analyzed cases will be kept.`,
+      content: `To add new data to an existing project, create a joint-called VCF and pedigree file with the new samples only. 
+      If you have new samples within a family that has been previously loaded you need all the previous family members in the new joint-called VCF. 
+      Upload it to seqr using the ‘Load Additional Data’ feature on the Project Page and select the new joint callset from the project’s associated workspace.`,
```

Comment:
I know this is the language in the ticket, but I think what we had before was better. Can we change it to the following and get an updated Spanish language translation for it @lynnpais 

```
To add new data, create a new joint called VCF with all the samples you want to include in your update. This should include any new samples you want to add to the project and any of their family memebers which have been previously loaded. Load this VCF using the Load Additional Data feature on the Project Page. All notes and tags saved in previously analyzed cases will be kept.
```